Library/Formula/aap.rb

```diff
@@ -0,0 +1,39 @@
+
+class Aap < Formula
+  homepage "http://www.a-a-p.org"
+  url "https://downloads.sourceforge.net/project/a-a-p/Aap/1.093/aap-1.093.zip"
+  sha256 "7a6c6c4a819a8379e60c679fe0c3f93eb1b74204cd7cc1c158263f4b34943001"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  # AAP installs "man" at top level. This moves it under share/.
+  patch :DATA
+
+  def install
+    # Aap is designed to install using itself
+    system "./aap", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    # We can't just do "aap --version" as a test, because it exits with an error status.
```

Comment:
No need to explain this. The test below is better than a simple `--version` test.